src/Symfony/Bridge/Doctrine/Tests/DependencyInjection/DoctrineExtensionTest.php

```diff
@@ -49,7 +49,7 @@ protected function setUp()
     }
 
     /**
-     * @expectedException LogicException
+     * @expectedException \LogicException
```

Comment:
expectedException always take a fqcn, which means there is no need for a starting `\` for these annotations